gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/ClouddriverService.java

```diff
@@ -46,7 +46,7 @@
   AccountDefinition updateAccountDefinition(@Body AccountDefinition accountDefinition);
 
   @DELETE("/credentials/{account}")
-  void deleteAccountDefinition(@Path("account") String account);
+  Response deleteAccountDefinition(@Path("account") String account);
```

Comment:
Isn't void correct given [this method signature](https://github.com/spinnaker/clouddriver/blob/d4be9ddfd4d82945fbadce504bf2aa7617fd0524/clouddriver-web/src/main/java/com/netflix/spinnaker/clouddriver/controllers/CredentialsController.java#L152)?

Reply:
Retrofit throws an error if the signature doesn't return anything.

Reply:
Aaah, ok.